SNSKY/donations.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- ticker: SNSKY
+    recipient:  
+    name: One Tree Planted
+    address: 145 Pine Haven Shores Rd, Shelburne, Vermont, 05482, US
+    tel: +1 800 408 7850
+    contact: hello@onetreeplanted.org
+  ada: 66
+  usd: 100
+  timestamp: 1621612800
+  files:
+    - SNSKY/2021_05_100_Trees.jpeg
+  charity_tx_id: 46-4664562
+  announcement_url: https://forest-fundraiser.raisely.com/sam
```

Comment:
Ah OK, there a confirmation is publically visible. I think that would be OK to also resolve the previous comment.
Still, if you have a payment confirmation please just add it. Thanks!

Reply:
yes..i have exactly for this reason started a fund raiser so that its public and no one needs to check whether my bills are no fake. i hope you can accept the fund raiser as proof